OsmAnd/src/net/osmand/aidl/AidlCallbackListener.java

```diff
@@ -0,0 +1,9 @@
+package net.osmand.aidl;
+
+import java.util.Map;
+
+public interface AidlCallbackListener{
+	long addAidlCallback(IOsmAndAidlCallback callback, int key);
```

Comment:
Interface needs documentation and proper indents between methods

Reply:
fixed & added